site/content/docs/5.2/getting-started/parcel.md

```diff
@@ -1,101 +1,152 @@
 ---
 layout: docs
-title: Parcel
-description: Learn how to include Bootstrap in your project using Parcel.
+title: "Bootstrap & Parcel"
+description: The official guide for how to include and bundle Bootstrap's CSS and JavaScript in your project using Parcel.
 group: getting-started
 toc: true
 ---
 
-## Install Parcel
+{{< callout >}}
+**Want to skip to the end?** Download the source code and working demo for this guide from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/parcel). You can also [open the example in StackBlitz](https://stackblitz.com/github/twbs/examples/tree/main/parcel?file=index.html) but not run it because Parcel isn't currently supported there.
+{{< /callout >}}
 
-Install [Parcel Bundler](https://parceljs.org/getting-started/webapp/).
+## Setup
 
-## Install Bootstrap
+We're building a Parcel project with Bootstrap from scratch, so there are some prerequisites and up front steps before we can really get started. This guide requires you to have Node.js installed and some familiarity with the terminal.
 
-[Install bootstrap]({{< docsref "/getting-started/download#npm" >}}) as a Node.js module using npm.
+1. **Create a project folder and setup npm.** We'll create the `my-project` folder and initialize npm with the `-y` argument to avoid it asking us all the interactive questions.
 
-Bootstrap depends on [Popper](https://popper.js.org/), which is specified in the `peerDependencies` property. This means that you will have to make sure to add both of them to your `package.json` using `npm install @popperjs/core`.
+   ```sh
+   mkdir my-project && cd my-project
+   npm init -y
+   ```
 
-When all will be completed, your project will be structured like this:
+2. **Install Parcel.** Unlike our Webpack guide, there's only a single build tool dependency here. Parcel will automatically install language transformers (like Sass) as it detects them.
+
+   ```sh
+   npm i --save-dev parcel
+   ```
+
+3. **Install Bootstrap.** Now we can install Bootstrap. We'll also install Popper since our dropdowns, popovers, and tooltips depend on it for their positioning. If you don't plan on using those components, you can omit Popper here.
+
+   ```sh
+   npm i --save bootstrap @popperjs/core
+   ```
+
+Now that we have all the necessary dependencies installed and setup, we can get to work creating the project files and importing Bootstrap.
```

Comment:
```suggestion
Now that we have all the necessary dependencies installed and set up, we can get to work creating the project files and importing Bootstrap.
```

Reply:
Not 100% sure. If correct, must be fixed in Webpack guide and in `customize/sass.md`